pulp_docker/app/viewsets.py

```diff
@@ -132,7 +132,10 @@ def create(self, request):
 
 class DockerRemoteViewSet(RemoteViewSet):
     """
-    A ViewSet for DockerRemote.
+    Docker remotes represent an external repository that implements the Docker
+    Registry API. Docker remotes support deferred downloading by gq can by
```

Comment:
What is "gq can"?

Reply:
gq is a vim alias reformat :) removed in the next push.